common/views/components/ExhibitionPromo/ExhibitionPromo.js

```diff
@@ -32,17 +32,17 @@ const ExhibitionPromo = ({
           ${spacing({s: 4}, {padding: ['bottom']})}
         `}>
         <p className={`no-padding ${spacing({s: 2}, {margin: ['bottom']})} ${font({s: 'HNM5'})}`}>
-          {end && 'Exhibition'}
-          {!end && 'Permanent exhibition'}
+          {format !== 'permanent' && 'Exhibition'}
```

Comment:
Should we make this more deliberate, i.e. have 'temporary' and 'permanent' as required types?

Reply:
We never surface `temporary` anywhere. Permanent exhibitions are the special snow flakes, so we mark them as such.

What value do we get from forcing temporary?

Reply:
This logic feels like it could definitely be simplified into:
```JS
{exhibition.format && `${exhibition.format.title} exhibition`}
{!exhibition.format && `Exhibition`}
```